code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm

```diff
@@ -61,6 +61,7 @@
 		for(var/mob/O in viewers(user, 3))
 			if((O.client && !( O.blinded )))
 				to_chat(O, "<span class='notice'>The locker has been [locked ? null : "un"]locked by [user].</span>")
+				playsound(src.loc, 'sound/machines/click.ogg', 15, 1, -3)

```

Comment:
I honestly don't think there is going to be that many of "that guy"'s that go around with a secure locker spamming the lock or spamming the locker in the middle of medbay or something.